server/src/pages/shot/shotpage-header.js

```diff
@@ -0,0 +1,187 @@
+/* globals controller */
+const React = require("react");
+const PropTypes = require("prop-types");
+const { Localized } = require("fluent-react/compat");
+const sendEvent = require("../../browser-send-event.js");
+const { SignInButton } = require("../../signin-button.js");
+const { Header } = require("../../header.js");
+const { TimeDiff } = require("./time-diff");
+
+exports.ShotPageHeader = class ShotPageHeader extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  renderMyShotsText() {
+    // FIXME: this means that on someone else's shot they won't see a My Shots link:
+    if (!this.props.isOwner) {
+      return (
+        <Localized id="shotPageBackToHomeButton">
+        <button className="nav-button" title="Homepage" onClick={this.onClickMyShots.bind(this)}>
+          <div className="back-to-home">
+            <h2>
+              Firefox
+            </h2>
+            <h3>
+              Screenshots
+            </h3>
+          </div>
+        </button>
```

Comment:
These links should use `<a>` and `hrefs`.  Buttons shouldn't have divs and  headings.